net/lp_rpc.proto

```diff
@@ -44,7 +44,7 @@ message OSInfo {
   enum StorageType {
     DIRECT     = 0;
     S3         = 1;
-    IPFS       = 2;
+    // IPFS       = 2;
```

Comment:
Do external packages depend on this enum?   Do we need to leave GOOGLE at the value of 3 for any reason?   Should we leave IPFS there even though it may not be used?

The golang idiomatic way of declaring enums is a little different that what we might be used to in c / c++ ..   I think this would be better to use `iota` if we're not locked into these integers for any reason.  See here:  https://stackoverflow.com/questions/14426366/what-is-an-idiomatic-way-of-representing-enums-in-go

Reply:
> Do external packages depend on this enum? 

not yet

>  Do we need to leave GOOGLE at the value of 3 for any reason?

no. I've changed it with next commit

> The golang idiomatic way of declaring enums

protobuf allows us to define enums, so it's better to use it

Reply:
Note that we should really never remove or renumber fields from a protobuf definition, because that breaks compatibility between different client versions. It's probably OK right now because I don't think we've actually made a release with any of this, but something to keep in mind as we go forward. (Might break nodes on the internal testnet though if anyone tries to use GCS...)

Reply:
> Note that we should really never remove or renumber fields from a protobuf definition, 

Yes, that's why initially I haven't changed it. But, given that it's actually haven't been deployed anywhere outside Livepeer, I think we can change it now.